Tests/PublishTests/Tests/DeploymentTests.swift

```diff
@@ -62,6 +62,8 @@ final class DeploymentTests: PublishTestCase {
 
         try shellOut(to: [
             "git init",
+            // Not all git installations init with a master branch.
+            "git checkout master || git checkout -b master",
```

Comment:
Many installations of git now create a `main` branch rather than a `master` one.